### ISSUE TITLE
Image doesn't preview when creating answer option

### ISSUE THREAD

**Issue by PatriciaMoller** [SmartZoos/Toolset#112]
After adding a picture to an answer option for a question, the image can't been seen while editing. It can been seen after saving and if the Activity Item is edited again.

**Comment by pjotrsavitski** [comment]
That one is intended, to be honest. We could show the preview of the image to be uploaded, same way as it is done during the gameplay. Does that make much sense, as the user knows which images is going to be uploaded.

Previous ones are a different matter.

**Comment by PatriciaMoller** [comment]
An instant preview is preferable. As it is now, I at first thought the picture had not been uploaded at all.

A preview, to see if the image looks good in the context, or some kind of feedback to indicate that the upload was successful would help.

**Comment by pjotrsavitski** [comment]
The upload only happens later, when the form is submitted. It is not a piece-by-piece update process.

**Comment by kaimikael** [comment]
I think this is a bit problematic.  We are after all working on a game – all interaction should get as immediate feedback as possible.  Instead of waiting for HTML to react to a Submit command, can we script the page to do the upload as soon as a file has been selected?

**Comment by pjotrsavitski** [comment]
@kaimikael Yes, although that would require major changes to the codebase. Especially with Activity Items that have not been created yet, options that have not been stored to the DB yet. But that is doable, just need to come up with suitable logic.

One more thing is with how all the other fields behave. If only images are stored at once and all others are stored on submit, then it could be confusing.

One more approach would be to show the small preview for the newly selected image. The way that work within the Game context. That could be resource-consuming on the Browser side (needs testing to say anything for sure).
